### PR TITLE
Contribution Guidelines

### PR DESCRIPTION
**Pull request by ixTec**
# Initial Contribution Guidelines

### ❓ Misc Changes

- Added initial version of contribution guidelines for review within team.

Feedback to be gathered and changes made before merging.

### CODE REVIEW

**Review by blurbyte — APPROVED** [review]
Looks amazing 🥇 ! Maybe mention Service Desk in 'I have a new component / style change / big feature' section. I would also capitalize or emphasize Slack and Service Deskt, etc.so they stand out a bit more.

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Consult our [Project Organization](https://github.com/datacamp/waffles/blob/master/docs/PROJECT_ORGANIZATION.md) and [Development Guidelines](https://github.com/datacamp/waffles/blob/master/docs/DEVELOPMENT.md) for more information about our project is structured and various aspects of it work.
+
+## I have a new component / style change / big feature
+
+Whilst we would love for everyone to be able to contribute new components or styling updates etc, these major changes require design sign off and heavy consideration and so you are best off reaching out to us so we can discuss it first.
+
+## I have a bug report / feature request / question
+
+We are more than happy to help! The best way to report these bugs or create requests is to ping our team on slack or raise an issue through service desk.
```
Comment (nunorafaelrocha) [inline docs/CONTRIBUTING.md:23]:
```suggestion
We are more than happy to help! The best way to report these bugs or create requests is to raise an issue through [service desk](https://datacamp.atlassian.net/servicedesk/customer/portal/1).
```

**Review by nunorafaelrocha — COMMENTED** [review]
I left a couple of comments 😃

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- **Follow our [Coding Style Guidelines](https://github.com/datacamp/waffles/blob/master/docs/CODING_STYLE.md)** for all changes.
+- **Branch from and target `rc-next`** for all contributions.
+  - Branch names should follow the `<user>/<branch-name>` format.
+- **Format commit messages** to follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification - please also provide meaningful commit messages (e.g. `refactor(button): clean up onclick handler`, not `refactor: handler`).
+- **Use the [PR Template](https://github.com/datacamp/waffles/blob/master/.github/PULL_REQUEST_TEMPLATE.md)** and fill out all applicable sections.
+
+Consult our [Project Organization](https://github.com/datacamp/waffles/blob/master/docs/PROJECT_ORGANIZATION.md) and [Development Guidelines](https://github.com/datacamp/waffles/blob/master/docs/DEVELOPMENT.md) for more information about our project is structured and various aspects of it work.
+
+## I have a new component / style change / big feature
```
[the remark this answers is not in the document]
Comment (ixTec) [reply docs/CONTRIBUTING.md:17]: Trying to differentiate between "Here's a PR of a change we want to make" and "I want you to make this change", hence the two sections

**Review by blurbyte — APPROVED** [review]

**Pull request merged**